ci/live-cluster-sanity.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+set -e
+cd "$(dirname "$0")/.."
+
+source ci/_
+source ci/rust-version.sh stable
+
+escaped_branch=$(echo "$BUILDKITE_BRANCH" | tr -c "[:alnum:]" - | sed -r "s#(^-*|-*head-*|-*$)##g")
```

Comment:
If `BUILDKITE_BRANCH` is empty (like if `ci/live-cluster-sanity.sh` is run locally), set `escaped_branch` to `$(whoami)` perhaps?

Reply:
done